yarn-project/sequencer-client/src/block_builder/solo_block_builder.ts

```diff
@@ -477,14 +477,7 @@ export class SoloBlockBuilder implements BlockBuilder {
     }
     const path = await this.db.getSiblingPath(treeId, index);
     // TODO: Check conversion from bigint to number
-    return new MembershipWitness(
-      height,
-      index,
-      assertLength(
-        path.data.map(b => Fr.fromBuffer(b)),
-        height,
-      ),
-    );
+    return new MembershipWitness(height, index, assertLength(path.toFieldArray(), height));
```

Comment:
Do you know about this todo? Not fully clear to me what conversion it is related to.

Reply:
Yep, it seems it doesn't apply anymore. IIRC there was an explicit `Number(bigintVariable)` here, but seems like it's gone (or moved elsewhere).